include/sliding_window/split/reference_segments.txt

```diff
@@ -0,0 +1,75 @@
+2243390
+0	2L 	0	2243390
```

Comment:
oh yeah, you are right this is pretty similar; we later just need to add the file name :)